hal/src/main/native/include/HAL/cpp/SerialHelper.h

```diff
@@ -19,14 +19,45 @@
 #include "HAL/SerialPort.h"
 
 namespace hal {
+/**
+ * A class for deterministically getting information about  Serial Ports.
+ */
 class SerialHelper {
  public:
   SerialHelper();
 
+  /**
+   * Get the VISA name of a serial port.
```

Comment:
Might be helpful to link to the NI documentation of what VISA is: http://www.ni.com/tutorial/3702/en/

Reply:
I'd rather not. Its not really something that should be used at all. Its just there because we use it in the HAL.

Reply:
What about saying NI-VISA instead of just VISA? At least then it's Google-able.